packages/ckeditor5-emoji/src/ui/emojipickerview.ts

```diff
@@ -0,0 +1,260 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module emoji/ui/emojipickerview
+ */
+
+import {
+	FocusCycler,
+	SearchInfoView,
+	View,
+	type FocusableView,
+	type ViewCollection,
+	type SearchTextViewSearchEvent
+} from 'ckeditor5/src/ui.js';
+import {
+	FocusTracker,
+	KeystrokeHandler,
+	type Locale,
+	type ObservableChangeEvent
+} from 'ckeditor5/src/utils.js';
+import EmojiGridView, { type EmojiSearchQueryCallback } from './emojigridview.js';
+import EmojiCategoriesView from './emojicategoriesview.js';
+import EmojiSearchView from './emojisearchview.js';
+import EmojiToneView from './emojitoneview.js';
+import type { SkinToneId } from '../emojiconfig.js';
+import type { EmojiCategory, SkinTone } from '../emojidatabase.js';
+
+/**
+ * A view that glues pieces of the emoji dropdown panel together.
```

Comment:
```suggestion
 * A view that glues pieces of the emoji balloon panel together.
```

Reply:
Or even without the _balloon_ word, as the view will also work if it will be inserted as a child of a dialog.

Reply:
Done in b53e60f9c094d5f821e1ee08726eb12038509a25.